documentatie/flowcharts/date-entry-navigation.md

```diff
@@ -0,0 +1,149 @@
+# Data Entry Navigation
+
+## Questions
+- Where does "cache input" happen in these flows?
+- Is there a different warning icon for accepted versus not-accepted warnings? Or is it not possible to leave a page without accepting the warning? Also, accepting warnings is not yet included in any of the flows.
```

Comment:
- it's impossible to leave the page without accepting the warning.
- there is no other icon. All warnings are accepted warnings and we show the icon regardless.